### PR TITLE
Updated Q&A after speaking with the PM

### PR DESCRIPTION
**Pull request by tweekerz**
Spoke with the PM and we edited this together

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-configuration-wizard-faqs.md [hunk 1]
```diff
@@ -28,15 +28,11 @@ To download the Office 365 Hybrid Configuration wizard go to [https://aka.ms/Hyb
...
-
-- **Exchange 2016**: The latest release of Exchange 2016 needs to be installed on at least one server running the Mailbox server role.
-
-For example, assume you've installed Exchange 2013 CU8 in your on-premises organization, and the latest available release of Exchange 2013 is CU10. To remain in a supported hybrid configuration, you need to upgrade your Exchange 2013 servers to at least CU9. However, we'd strongly recommend you upgrade to CU10.
+> A: You can use a combination of one or more of the following:
+>
+> Exchange 2010 SP3 (latest RU strongly recommended), Exchange 2013 through Exchange 2019 (where applicable, latest CU strongly recommended). Important note: To remain in a supported hybrid configuration, you need to make sure you are running the latest available released CU for your Exchange version. If you are unable to upgrade to the latest available CU, the previous version is also supported.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-configuration-wizard-faqs.md]:
```suggestion
> Exchange 2010 SP3 (latest RU strongly recommended), Exchange 2013 through Exchange 2019 (where applicable, latest CU strongly recommended). To remain in a supported hybrid configuration, you need to make sure that you are running the latest available released CU for your Exchange version. If you are unable to upgrade to the latest available CU, the previous version is also supported.
```
Please remove "Important note". Alternatively, the explanation should be inserted as a note in note format.

File: Exchange/ExchangeHybrid/hybrid-configuration-wizard-faqs.md [hunk 2]
```diff
@@ -28,15 +28,11 @@ To download the Office 365 Hybrid Configuration wizard go to [https://aka.ms/Hyb
...
+>
+> For role specific requirements and prerequisites please see [https://docs.microsoft.com/en-gb/exchange/hybrid-deployment-prerequisites#prerequisites-for-hybrid-deployment](https://docs.microsoft.com/en-gb/exchange/hybrid-deployment-prerequisites#prerequisites-for-hybrid-deployment)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-configuration-wizard-faqs.md]:
```suggestion
> For role specific requirements and prerequisites, please see [https://docs.microsoft.com/exchange/hybrid-deployment-prerequisites#prerequisites-for-hybrid-deployment](https://docs.microsoft.com/exchange/hybrid-deployment-prerequisites#prerequisites-for-hybrid-deployment).
```

File: Exchange/ExchangeHybrid/hybrid-configuration-wizard-faqs.md (hunk 1 above)
Comment (nenonix) [inline Exchange/ExchangeHybrid/hybrid-configuration-wizard-faqs.md]:
```suggestion
> Exchange 2010 SP3 (latest Update Rollup strongly recommended), Exchange 2013 through Exchange 2019 (where applicable, latest CU strongly recommended). 
>
> [!IMPORTANT]
> To remain in a supported hybrid configuration, you need to make sure you are running the latest available released CU for your Exchange version. If you are unable to upgrade to the latest available CU, the previous version is also supported.
```

**Pull request merged**